bots/images/scripts/openshift.setup

```diff
@@ -314,6 +314,8 @@ printf 'AuthorizedKeysCommand /usr/local/bin/authorized-kube-keys --kubeconfig=/
 # Prepare Kubevirt for later installation
 /var/lib/testvm/kubevirt.setup
 
+oc project default
```

Comment:
Can we please leave this out of the images? If this is necessary at all, it should be done in the tests. IMHO it's better to not pre-apply such state to the built images, as they are much harder to change than the tests.

Reply:
Right, added after the last build. The `kubevirt.setup` explicitly switches the project to `kubevirt` and leaving it as it is. It means, when the `openshift` image boots, the `kubevirt` project is the actual one breaking former test flow.
I can "clean-up" in `kubevirt.setup`. But making it here is less-error prone for the future and making the state more obvious for others.

Reply:
Ah, that makes sense. Thanks!

Reply:
I locally moved it into kubevirt.setup, and into the "kubevirt.setup: Adjust to 0.4.1" commit with a comment, so that it's easier to see why that is done.